code/obj/item/device/pda2/smallprogs.dm

```diff
@@ -1620,3 +1620,27 @@ Using electronic "Detomatix" SELF-DESTRUCT program is perhaps less simple!<br>
 		signal.data["sender"] = src.master.net_id
 
 		SEND_SIGNAL(src.master, COMSIG_MOVABLE_POST_RADIO_PACKET, signal, null, "power_control")
+
+//Genebooth Tracker
+/datum/computer/file/pda_program/genebooth_tracker
+	name = "Genebooth Tracker"
+	size = 6
+
+	return_text()
+		if(..())
+			return
+
+		. = src.return_text_header()
+
+		var/booth_counter = 0
+		for_by_tcl(GB, /obj/machinery/genetics_booth)
```

Comment:
```suggestion
		for_by_tcl(booth, /obj/machinery/genetics_booth)
```
try not to use non-standard 1 or 2 letter variable names (ie `turf/T` is okay, but `genetics_booth/GB` is not)